tof_node:
  ros__parameters:
    tof_model: vl53l0x
    tof_fov_x: 25.0 # deg
    tof_fov_y: 25.0 # deg
    tof_range_z_min: 0.03 # m
    tof_range_z_max: 2.0 # m
    tof_histogram: false
